forc-plugins/forc-client/src/util/tx.rs

```diff
@@ -156,7 +160,9 @@ impl<Tx: Buildable + SerializableVec + field::Witnesses + Send> TransactionBuild
                         let accepted = ask_user_yes_no_question(&question)?;
                         if accepted {
                             new_wallet_cli(&wallet_path)?;
-                            println!("Wallet created successfully.")
+                            println!("Wallet created successfully. Deriving first account");
```

Comment:
```suggestion
                            println!("Wallet created successfully. Setting up your first account...");
```
Perhaps this language is clearer?

Reply:
That is a good point to discuss. Because we use this terminology all around the stack. IMO suggestion makes it clear for most of the users but I believe technically correct term is "deriving" so I am perfectly happy to apply the suggestion or leave this as it is.


### A little explanation of how I mapped this stuff in my mind

This terminology is coming from the fact that forc-wallet is a HD-wallet, ([here](https://www.ledger.com/academy/crypto/what-are-hierarchical-deterministic-hd-wallets) is a nice article about what it is) and with hd-wallet we derive accounts. (A really simplified summary of HD wallet's usage of "derivation" comes from the fact that accounts already exist all the time but not discovered by the user yet, thus they are derived using your private key)

Maybe developer facing stuff should use the right technical term but stuff like front-end wallet should mention this as setting up? Anyways let me know what you think and we can apply this suggestion to couple more places derivation is used for account creation.

Reply:
Is "derive" used in any other user-facing contexts at the moment?

Reply:
Hmm i though we used that more frequently but quick search yielded only the https://github.com/FuelLabs/forc-wallet/blob/master/README.md, might need to look deeper to find all occurrences.

Reply:
Thanks for the explanation Kaya and for the link. In that case I think deriving is more accurately describing what is going on. Happy for us to continue using that terminology.